spec/system/smoke_spec.rb

```diff
@@ -6,14 +6,14 @@
 require "site_prism"
 require "site_prism/all_there"
 
-require "support/page_objects/eligiblity_interface/country"
-require "support/page_objects/eligiblity_interface/degree"
-require "support/page_objects/eligiblity_interface/eligible"
-require "support/page_objects/eligiblity_interface/misconduct"
-require "support/page_objects/eligiblity_interface/qualification"
-require "support/page_objects/eligiblity_interface/region"
-require "support/page_objects/eligiblity_interface/start"
-require "support/page_objects/eligiblity_interface/teach_children"
+require "support/autoload/page_objects/eligibility_interface/country"
+require "support/autoload/page_objects/eligibility_interface/degree"
+require "support/autoload/page_objects/eligibility_interface/eligible"
+require "support/autoload/page_objects/eligibility_interface/misconduct"
+require "support/autoload/page_objects/eligibility_interface/qualification"
+require "support/autoload/page_objects/eligibility_interface/region"
+require "support/autoload/page_objects/eligibility_interface/start"
+require "support/autoload/page_objects/eligibility_interface/teach_children"
```

Comment:
Hmm is this going to cause problems because it'll try and use the radio button page objects, but they won't have been autoloaded in the smoke test environment 🤔

Reply:
Yeah. I was going to remove all of those altogether but I'm not sure what Rails.env they run in. I've only autoloaded everything in test.

Reply:
They don't run in a Rails environment at all which makes them a bit difficult to work with - because there's no database available trying to load Rails fails.

It's the reason I had to revert https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/408 and had to put the `require` in place originally: https://github.com/DFE-Digital/apply-for-qualified-teacher-status/commit/5843662aa36645546670de54b7205461ec4fb093

Reply:
I've added the missing stuff. They're working locally for me now.